doc_src/gear_developers/limitation.md

```diff
@@ -0,0 +1,12 @@
+# Limitation
+
+There are certain limitations in Antiktyera.
+We introduce some main limitations. But all limitations are not described here. Please also read other parts of the document when you implement gear.
+
+* There is an upper limit of the execution time of web/g2g request
```

Comment:
```suggestion
* There is an upper limit of the execution time of web/g2g request.
```

Reply:
I agree on it.

Reply:
I fixed in 2b732f861c322c0128675ac8847d6e2707104592